src/FloorSubsystem/FloorSystem.java

```diff
@@ -61,7 +68,12 @@ public void run() {
 			
 			try {
 				Thread.sleep((long) (currentTime - timeZero));
```

Comment:
This line makes me think we are already waiting the correct amount of time between when we send out messages from the floor to the scheduler. Is this correct? If not, we still have to add that functionality.